hosting/src/app/games/quiz/lobby/lobby.component.html

```diff
@@ -11,7 +11,11 @@ <h2>Create Question</h2>
 
                         <brand-dropdown label="Type" placeholder="Select an option" formControlName="type" [options]="availableTypes"></brand-dropdown>
 
-                        <brand-input label="Question" type="textarea" formControlName="text"></brand-input>
+                        <span *ngIf="questionForm.get('type').value.id =='image'; else standardQuestion">
+                            <brand-label description="(image should include any text for the question)">Image URL</brand-label>
+                            <brand-input   type="text" formControlName="text" placeholder="http://url-of-the-image"></brand-input>
+                        </span>
+                        <ng-template #standardQuestion><brand-input  label="Question" type="textarea" formControlName="text"></brand-input></ng-template>
```

Comment:
Another alternative could be to use the form type attribute as the input type, for sudo example

```typescript
 [type]="getType()"
```